parlai/agents/transformer/transformer.py

```diff
@@ -166,7 +166,7 @@ def add_cmdline_args(cls, argparser):
             '--reduction-type',
             type=str,
             default='mean',
-            choices=['first', 'max', 'mean'],
+            choices=['first', 'max', 'mean', 'none'],
             help='Type of reduction at the end of transformer',
```

Comment:
Should this really be in there?

Reply:
we check for it in modules https://github.com/facebookresearch/ParlAI/blob/45b7a9565ce980cad3f29e4747defa66fef96024/parlai/agents/transformer/modules.py#L480

but we can't actual invoke this option since there is a list of choices and a default